src/main/java/net/glowstone/GlowServer.java

```diff
@@ -1338,6 +1335,20 @@ public int broadcastMessage(String message) {
     }
 
     @Override
+    public void broadcast(BaseComponent component) {
+        for (Player player : getOnlinePlayers()) {
+            player.spigot().sendMessage(component);
+        }
+    }
+
+    @Override
+    public void broadcast(BaseComponent... components) {
```

Comment:
The only reason I could think of is, that If I combine these BaseComponents and send it to every player, it could be faster.